src/index.ts

```diff
@@ -0,0 +1,61 @@
+import * as express from 'express';
+import * as Realm from 'realm';
+
+const app = express();
+const port = 3500;
+
+const CarSchema = {
```

Comment:
- Use different file for feature related work

Reply:
Okay!